File: TicketingSystem/ticketing-system/src/app/api/models/seat-header.ts
/* tslint:disable */
/* eslint-disable */
import { GameSeatHeader } from './game-seat-header';
export interface SeatHeader {
  gameSeats?: null | Array<GameSeatHeader>;
  id?: number;
  rowId?: number;
  rowNumber?: number;
  seatNumber?: number;
  sectorName?: null | string;
  status?: null | string;
}

File: TicketingSystemAPI/TicketingSystemBLL/Dtos/SeatHeader.cs
using TicketingSystemDB.Entities;

namespace TicketingSystemBLL.Dtos
{
    public class SeatHeader
    {
        public SeatHeader()
        {
            GameSeats = new List<GameSeatHeader>();
        }

        public int Id { get; set; }
        public int SeatNumber { get; set; }

        public int RowId { get; set; }

        public int RowNumber { get; set; }

        public string SectorName { get; set; } = null!;

        public string Status { get; set; } = null!;

        public virtual List<GameSeatHeader> GameSeats { get; set; }

    }
}
